torch/csrc/stable/parallel_utils.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <torch/csrc/inductor/aoti_torch/c/shim.h>
```

Comment:
I'm somewhat new to the codebase, but this header name makes me a bit uncofortable. Are stable API simply piggy backing on some of AOTI shim definitions? And is the intention to move it later into torch/csrc/stable folder?